src/assets/json/checklist/tests-web-checklist-en.json

```diff
@@ -0,0 +1,155 @@
+	"profils": "",
+	"wcag": ["1.4.3"],
+	"ID": "testWebID-16",
+	"IDorigin": "testWebID-16"
+}, { 
+	"themes": "Colors and contrasts",
+	"title": "Make sure the color is not used as the only means of conveying information",
+	"type": ["Manual test"],
+	"tests": ["Identify elements using color to convey information"],
+	"verifier": ["Color is not the only means of conveying information: at least one other visual method is available to obtain the same information."], 
+    "resultat": ["Information conveyed by color can also be obtained by means of an explicit text", "The information conveyed by color is complemented by other visual information (e.g., icons use different colors, but also different shapes).", "Special case of links in text: if they are not underlined, provide a means other than color to distinguish them on keyboard focus and mouse hover."], 
+    "exception": "", 
+	"raccourcis": "",
+	"moreInfo": "https://a11y-guidelines.orange.com/en/web/develop/colors-and-contrasts/#do-not-use-colour-or-sensory-characteristics-as-the-unique-source-of-information",
+	"profils": "",
+	"wcag": ["1.4.1"],
+	"ID": "testWebID-18",
+	"IDorigin": "testWebID-18"
+}, {
+	"themes": "Common navigation",
+	"title": "Allow control of animations", 
+    "type": ["Manual test"], 
+    "tests": ["Identify any content that is moving, auto-updated, flashing, or scrolling for more than 5 seconds and starts automatically (for example a carousel)."], 
+    "verifier": ["For each content, a mechanism is present to allow the user to pause it or hide it.", "This mechanism is accessible with mouse and keyboard navigation."], 
+    "resultat": ["User can pause or hide animations, movements, updates or blinks."], 
+    "exception": "", 
+	"raccourcis": "",
+	"moreInfo": "https://a11y-guidelines.orange.com/en/web/develop/common-navigation/#allowing-animations-control",
+	"profils": "",
+	"wcag": ["2.2.2"],
+	"ID": "testWebID-37",
+	"IDorigin": "testWebID-37"
+}, {
+	"themes": "Keyboard navigation",
+	"title": "Allow the use of the application with keyboard",
+	"type": ["Keyboard navigation"],
+	"tests": ["Navigate through the page with the keyboard using the <kbd>shift</kbd> + <kbd>tab</kbd> keys.","Use all interactive elements (by pressing the <kbd>Enter</kbd> and <kbd>Space</kbd> keys for buttons/links, and the arrow keys for some components: a series of radio buttons, a tab system, etc.)."], 
+	"verifier": [["Everything that can be done with the mouse can be done using the keyboard and vice-versa"],["The focus movement order maintains a logical reading and usage flow."],["The focus does not get stuck on a part of the page (keyboard trap)."],["The focus indicator (outline) is visible on all interactive elements and has sufficient contrast."]],
+	"resultat": ["All interactive elements can be reached from a keyboard navigation", "All interactive elements are usable with keyboard interactions"],
```

Comment:
Not sure why sometimes you a add a '.' at the end of a list item and sometimes not. In some list like this one, some `li` end by a '.' and some not.  I honestly don't really care but it should be consistent in the page (maybe check the french version too).